api/src/main/java/jakarta/data/repository/Find.java

```diff
@@ -26,11 +26,25 @@
 
 
 /**
- * <p>Annotates a repository method returning entities as a parameter-based automatic query method.</p>
+ * <p>Annotates a repository method to be a parameter-based automatic query method
```

Comment:
Nitpick, but I find "Annotates X to be Y" really clumsy English.

It's fine to write "Declares X to be Y", but I think "Annotates X as Y" was also perfectly OK.

Reply:
It looks like I rewrote that sentence when switching around two parts of it. I'll get that corrected back to "as" instead of "to be"

Reply:
Corrected in https://github.com/jakartaee/data/pull/921/commits/d93b48e148052c18563811fee1c931e0a6eff451 along with removal of an extra "a" that I spotted in a different section that I had updated.